src/hostname/hostnamed.c

```diff
@@ -430,6 +430,32 @@ static int context_write_data_machine_info(Context *c) {
         return write_env_file_label("/etc/machine-info", l);
 }
 
+// https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/sound/soc/soc-core.c?h=v5.10#n1476
+static const char * const dmi_blocklist[] = {
+        "To be filled by OEM",
+        "TBD by OEM",
+        "Default String",
+        "Board Manufacturer",
+        "Board Vendor Name",
+        "Board Product Name",
+        "N/A",
+        NULL,
+};
```

Comment:
i'd prefer if we did this via hwdb too. i.e. just add hwdb entries matching these strings that assign the attribute prop you added the empty string, so that nothing matches.